source/contribute/index.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Contribute
+github_link: contribute/index.html
+indexed: true
+---
+
+## Contribute to Shopware
+
+Shopware is available under dual license (AGPL v3 and proprietary license). If you want to contribute code (features or bugfixes), you have to create a pull request and include valid license information. You can either contribute your code under New BSD or MIT license.
+
+If you want to contribute to the backend part of Shopware, and your changes affect or are based on ExtJS code, they must be licensed under GPL V3, as per license requirements from Sencha Inc.
+
+If you are not sure which license to use, or want more details about available licensing or the contribution agreements we offer, you can contact us at <contact@shopware.com>.
+
+- **Contribution guide**: <https://github.com/shopware/shopware/blob/master/CONTRIBUTING.md>
+- **License**: Dual license AGPL v3 / Proprietary
+- **Github Repository**: <https://github.com/shopware/shopware>
+- **Issue Tracker**: <http://jira.shopware.de/jira>
+
+## Contribute to the Dev Docs
+
+These doc pages are also available on Github and open to contribution from the community. 
+
+- **Github Repository**: <https://github.com/shopware/devdocs>
+- **License**: Proprietary
+
+Contributions to this project should aim at improving the overall coverage / quality of the Shopware 5 documentation page.
+ 
+All contributions should respect the following styling guidelines.
+
+### Language
+
+All documents should be written in American or UK English. Should it be necessary, disambiguation between the two should be done on a per occurrence basis, giving preference to the one most commonly used on similar contexts (other documentation sites, for example)
+
+### Naming conventions
+
+Products/libraries/technologies should start with capital letters, unless explicitly specified so by the author/owner. Acronyms should be full upper case. Examples of correct usage:
+
+- Shopware (the product)
+- Less
+- CSS
+- JavaScript
+- ID
+- HTML
+- HTTP/HTTPS
+- SSL
+- Responsive/Base (for themes names)
+- shopware AG (company name)
+
+When referring to specific parts of Shopware's interface, like buttons or modules, the naming should be an exact match to what users expect to find in the actual code/interface. Examples:
```

Comment:
And that's why we need a rule for it. Otherwise we will have "articles" on half of the code/docs, and "products" on the other half. Loss in consistency means potential problems for new users who are not so familiar with Shopware or our double naming. I vote for keeping this